packages/renderer/src/lib/dashboard/NotificationCardItem.svelte

```diff
@@ -11,35 +11,58 @@ export let notification: NotificationCard;
 async function removeNotification(id: number): Promise<void> {
   await window.removeNotification(id);
 }
+
+const notificationStyleMap = {
+  info: {
+    borderColor: 'border-[var(--pd-state-info)]',
+    iconColor: 'text-[var(--pd-state-info)]',
+    icon: faCircleInfo,
+  },
+  warn: {
+    borderColor: 'border-[var(--pd-state-warning)]',
+    iconColor: 'text-[var(--pd-state-warning)]',
+    icon: faExclamationTriangle,
+  },
+  error: {
+    borderColor: 'border-[var(--pd-state-error)]',
+    iconColor: 'text-[var(--pd-state-error)]',
+    icon: faCircleExclamation,
+  },
+};
+
+const { [notification.type]: notificationStyle } = notificationStyleMap;
 </script>
 
 <div
-  class="relative bg-[var(--pd-content-card-carousel-card-bg)] w-full py-4 px-5 rounded-md"
+  class="bg-[var(--pd-content-card-bg)] text-[var(--pd-content-card-text)] p-4 w-full {notificationStyle.borderColor} border-t-[3px]"
   role="region"
   aria-label="id: {notification.id}">
-  <div class="flex flex-row">
-    <div class="mr-3">
-      {#if notification.type === 'info'}
-        <Fa size="1.5x" class="text-[var(--pd-notification-dot)]" icon={faCircleInfo} />
-      {/if}
+  <div class="flex flex-row space-x-3">
+    <div class="flex">
+        <Icon size="1.5x" icon={notificationStyle.icon} class={notificationStyle.iconColor} />
     </div>
-    <div class="flex flex-col space-y-2">
-      <div class="text-[var(--pd-content-card-carousel-card-header-text)] font-bold" aria-label="Notification title">
-        {notification.title}
+    <div class="flex flex-col space-y-2 flex-1">
+      <div class="flex flex-row items-center justify-between">
+        <span class="font-medium" aria-label="Notification title">
+          {notification.title}
+        </span>
       </div>
-      <div class="text-[var(--pd-content-card-carousel-card-text)]" aria-label="Notification description">
+      <span aria-label="Notification description">
```

Comment:
Suggestion: 

Hide this span when notification does not have a body. Otherwise it leaves a 8px margin underneath the title and the notification text is not vertically centered.

```
{#if notification.body}
<span aria-label="Notification description">
    <Markdown markdown={notification.body} />
</span>
{/if}
```
Before:
<img width="966" height="1056" alt="Screenshot 2025-07-25 at 11 40 54" src="https://github.com/user-attachments/assets/3ce20e5e-926d-48cf-b8a4-9ae0b4006f16" />

After: 
<img width="1010" height="1100" alt="Screenshot 2025-07-25 at 11 46 29" src="https://github.com/user-attachments/assets/947d794e-e2d0-4b90-ab56-7913f671fb00" />